src/Helper/Input/Checkbox.php

```diff
@@ -17,6 +17,33 @@
  */
 class Checkbox extends AbstractChecked
 {
+
+    /**
+     *
+     * Individual attributes for each option (only for multi checkbox)
+     *
+     * @var array
+     *
+     */
+    protected $optionsAttribs = array();
```

Comment:
There is also a dilemma in this naming. If I am correct in v2 we have used underscores. So there is a consistency issue I guess. We can revisit this later.